phpspec.yml.dist

```diff
@@ -2,3 +2,21 @@ suites:
     sylius_refund:
         namespace: Sylius\RefundPlugin
         psr4_prefix: Sylius\RefundPlugin
+
+extensions:
+    LeanPHP\PhpSpec\CodeCoverage\CodeCoverageExtension:
+        blacklist:
```

Comment:
I don't like the blacklist thing here, I would rather see what classes are not covered by PHPSpec rather than redefining them here. Could we also get PHPUnit coverage and merge them together? It seems a bit problematic for contributors, how would I know whether I should put some of my files here when I add a new feature?

Reply:
I was thinking about such an approach, but I believe there are some few strong pros for a blacklist.

This list contains a list of catalogs we **for sure** don't want to cover by specs and we will not do it in any future (I think we agree it's unnecessary for controllers or dependency injection configuration). I see this blacklist as a configuration perfectly fit to specify such catalogs. Moreover, if we just allow them to be included in a code coverage report, it would be much harder to notice if any future PR does not contain the required specs (e.g. 100%->98.9% is more visible than 91.95%->90.85%). 

I agree that it can be not clear for contributors to determine "does my new class require PHPSpec test or not". Therefore, I think it should be clearly said in a proper README section when we should use a PHPSpec. In my opinion, classes that should be designed with PHPSpec are all services that contain any **logic**, any describable **behavior** - none of the blacklisted classes contain one, they're just configuration or raw data. We can also advice in a documentation "if you're not sure about writing a spec - write it". All mistakes (unneeded spec file, or missing test in an excluded directory) should be easily catchable during the code review process 👨‍💻 

Regarding PHPUnit - I don't know, I'll check it 😄 I'm not sure how it will behave, as we have only one `TemporaryFileManagerTest` file. But if it works with no problem, I'll add it :)